packages/default/scss/postcss.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
All themes need to share the same PostCSS config. Perhaps the config can be shared by exporting it from the themes-build package?

Reply:
Will try to do it like that .